finviz/screener.py

```diff
@@ -13,6 +13,48 @@
 class Screener(object):
     """ Used to download data from http://www.finviz.com/screener.ashx. """
 
+    @classmethod
+    def from_url(cls, url, rows=None):
+        splitted_url = urlparse(url)
+        splitted_query = urlparse_qs(splitted_url.query)
+
+        tickers = None
+        if 't' in splitted_query:
+            tickers = splitted_query['t'][0].split(',')
+
+        filters = None
+        if 'f' in splitted_query:
+            filters = splitted_query['f'][0].split(',')
+
+        table = None
+        if 'v' in splitted_query:
+            table_numbers_types = {
```

Comment:
`table_numbers_types` is the same variable as `self._table_types` you should avoid variable duplication (code smell).

Reply:
`table_numbers_types` is to get the table_type from the number. `self._table_types` is to get the number from the table_type. Their purpose is reversed. Also, `self._table_types` is an object attribute only accessible after object initialization, so inaccessible to the classmethod `from_url` which is executed before object initialization.

To avoid the duplication, one way to do it is define `table_types` as a class variable or outside of Screener. So to get table_numbers_types from table_types, it can be done this way: 
```python
table_numbers_types = {v: k for k, v in table_types.items()}
```
There will be no code duplication here